tests/hybrid/test_dispatch.py

```diff
@@ -637,6 +657,23 @@ def test_hybrid_dispatch_one_cycle_heuristic(site):
     hybrid_plant.simulate(1)
 
     assert sum(hybrid_plant.battery.Outputs.P) < 0.0
+
+
+def test_hybrid_dispatch_one_cycle_heuristic_baseload(site):
```

Comment:
Need to add test for the other control analysis method

Reply:
I think both are tested, one in each of these two tests